client/src/components/NewsArticle/index.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import './style.scss';
+import { Layout } from '../Layout';
+import { fetchNewsData } from '../../containers/NewsLogic/action';
+import { connect } from 'react-redux';
+import Loader from 'react-loader-spinner';
+
+
+class NewsArticle extends Component {
+  componentDidMount() {
+    const { fetchNewsData } = this.props;
+
+    fetchNewsData();
+  }
+  
+  shortText(text) {
+    return text.length < 100 ? text
+      : <h2>{text.slice(0, 80)}...</h2>;
+  };
+
+  render() {
+    const { newsData, match } = this.props;
+
+    const { id } = match.params;
+    const art = newsData.find(art => id === art._id);
+
+    return (
```

Comment:
Am I right, if no data -- you will show loader? If I am right, how much time it will be on the page?

Reply:
when Roman finishes 404 i will change it